dev-libs/optix/optix-7.1.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="NVIDIA Ray Tracing Engine"
+HOMEPAGE="https://developer.nvidia.com/optix"
+SRC_URI="NVIDIA-OptiX-SDK-${PV}-linux64-x86_64.sh"
+
+SLOT="0/7"
+KEYWORDS="~amd64"
+RESTRICT="fetch"
+LICENSE="NVIDIA-r2"
+
+RDEPEND="dev-util/nvidia-cuda-toolkit
+	virtual/opengl
+	media-libs/freeglut"
+DEPEND="${RDEPEND}"
```

Comment:
Remove `DEPEND`, installation is just an extraction, no need for any libraries.

Reply:
Agreed - when I first poked at this, I thought the SDK needed to be compiled. Then I discovered the installer doesn't compile it itself, so why should we?

Reply:
Sorry, maybe I was unclear.
Keep the `RDEPENDS`, only remove the `DEPENDS`.
This is important for cross compilation.

Reply:
Oh! Putting them back now...

Reply:
RDEPENDS = packages needed during running time
DEPEND = packages needed during linking time
BDEPEND = packages needed to be **run** to compile.